dts/bindings/pwm/nxp,lpc-pwm.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2017, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP LPC PWM
+
+compatible: "nxp,lpc-pwm"
+
+include: [pwm-controller.yaml, base.yaml]
+
+properties:
+    prescaler:
+      type: int
+      required: false
+      default: 1
```

Comment:
All uses of default must be justified in the description as documented in https://docs.zephyrproject.org/latest/guides/dts/bindings.html#default-values-for-properties.

Reply:
Thank you, I have updated the property description